### PR TITLE
Proposal for arrays

### PR DESCRIPTION
**Pull request by Mrxx99**
Some ideas for arrays. Suggestions for better english wording is welcome.
I also could propably add more examples.

### CODE REVIEW

**Comment by Mrxx99** [comment]
Seems like I forgot the set indexer, will update this PR tomorrow

**Comment by Mrxx99** [comment]
Somehow I missed #38. I actually like most of it better than my proposal, so will update this

**Comment by N-Masi** [comment]
Glad you liked my ideas!

File: README.md [hunk 1]
```diff
@@ -130,6 +130,23 @@ A poetic number literal begins with a variable name, followed by the keyword `is
...
+
+To add to an existing array use: `{value} belongs to {array}`
+
+To get an item from an array use: `get {index} from {array}` (of course indexes start at 0)
+An alias for `get` is `give me`
+Note: The poetic numbers can also be used for indexing
+
+`Get love from You` will give the 4th elemt of the array `You`.
+
+To get the length of the array use: `How long {array} is`
```
Comment (ignasi35) [inline README.md]: An alias for `How long` is `How deep`.

**Comment by crusaderky** [comment]
See my comment on #38 on why I think this isn't a good idea

**Comment by Mrxx99** [comment]
included suggestions of review and updated based on #38

**Comment by cwfitzgerald** [comment]
I have one major and one minor complaint with this PR. The major problem is `Put 123 and 234 into Array` is ambiguous. Either it means `Array = [123, 234]` or it means `Array = 123 && 234` i.e `Array = true` Because this happens at a higher level of the grammar, this will be totally ambiguous.

**Comment by dylanbeattie** [comment]
Arrays and stack operations are in the main Rockstar spec now, incorporating most of the discussion (if not necessarily the syntax) from this and linked PRs.

**Pull request closed without merging**